### PR TITLE
[DESIGN2-203-closeButton] closeButton Component - DSCO - Update color variables

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with Primitive colors from `primitiveColors.css` on the closeButton component. 

👀 **DOTD note:** there might be Eyes diffs with this.

## Links
Jira ticket: [DESIGN2-203](https://codedotorg.atlassian.net/browse/DESIGN2-203)

## Testing story
Tested locally in Storybook

----

## Before - using `shared/colors.scss`
Before

## After - using `primitiveColors.css`
After

### CODE REVIEW

File: apps/src/componentLibrary/closeButton/closeButton.module.scss [hunk 1]
```diff
@@ -28,7 +28,7 @@
   }
 
   &:focus-visible {
-    outline: 2px solid $light_primary_500;
+    outline: 2px solid var(--brand-teal-50);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/closeButton/closeButton.module.scss:31]: teal

File: apps/src/componentLibrary/closeButton/closeButton.module.scss [hunk 2]
```diff
@@ -41,26 +41,26 @@
 
 // Button Colors
 .closeButton-dark {
-  color: $light_gray_600;
+  color: var(--neutral-gray-60);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/closeButton/closeButton.module.scss:44]: dark

File: apps/src/componentLibrary/closeButton/closeButton.module.scss [hunk 3]
```diff
@@ -41,26 +41,26 @@
...
 
   &:active {
-    color: $light_gray_600;
+    color: var(--neutral-gray-60);
   }
 }
 
 .closeButton-light {
-  color: $light_gray_400;
+  color: var(--neutral-gray-40);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/closeButton/closeButton.module.scss:56]: light

**Review by levadadenys — APPROVED** [review]
LGTM!

**Pull request merged**